framework_tvb/tvb/core/services/burst_service.py

```diff
@@ -28,18 +28,24 @@
 #
 #
 
+import os
 from datetime import datetime
+from tvb.adapters.datatypes.h5.burst_configuration_h5 import BurstConfigurationH5
```

Comment:
This is not a nice added dependency (from core towards adapters).
Could we move `BurstConfigurationH5 ` under `tvb.core.entities.file` somewhere?